src/main/java/edu/suffolk/litlab/efspserver/SendMessage.java

```diff
@@ -1,11 +1,88 @@
 package edu.suffolk.litlab.efspserver;
 
-import com.twilio.*;
+import java.io.IOException;
+import java.util.HashMap;
+import com.twilio.Twilio;
+import com.twilio.rest.api.v2010.account.Message;
+import com.twilio.type.PhoneNumber;
 import com.sendgrid.*;
+import com.sendgrid.helpers.mail.Mail;
+import com.sendgrid.helpers.mail.objects.Content;
+import com.sendgrid.helpers.mail.objects.Email;
+import com.hubspot.jinjava.Jinjava;
 
 
 public class SendMessage {
+
+  public static void main(String[] args) throws IOException {
+    HashMap<String, Object> context = new HashMap();
+    context.put("name", "Suffolk LIT Lab Test");
+    // SendEmail("test@example.com", "Test email", "massaccess@suffolk.edu", "Hello, {{ name }}. message body", context);
+
+    //SendSMS("+1.........", "Hello {{ name }}", context);
+  }
   
+  /**
+   * Send an email, currently using the SendGrid API. Depends on the environment variable SENDGRID_API_KEY
+   * which should be a valid API key for Twilio SendGrid.
+   * @param from The email "from" address
+   * @param subject The email subject
+   * @param to The email recipient
+   * @param message_template A string with optional Jinja2 template variables. Will be rendered as plain text.
```

Comment:
Is it worth it to be able to send HTML messages/should that be a thing that callers can control? Not sure the range on what's supported in email HTML clients, but some branding + logos would probably be nice.